JenkinsfileProductionRelease

```diff
@@ -36,8 +36,7 @@ node('heavy && linux && docker') {
         withEnv([
           'SFDX_AUTH_JWT_USERNAME=sfdc.integration.devv2.hub@coveo.com'
         ]) {
-          sh 'npm run build'
```

Comment:
Guessing this removal is the main functional change and the rest is refactor?

Reply:
Yes exactly. Nx will automatically build only what's necessary.